guide/creating-your-bot/adding-more-commands.md

```diff
@@ -16,7 +16,7 @@ client.on('message', message => {
 });
 ```
 
-Before doing anything else, make a property to store the prefix you've configured. Instead of `const config = ...`, you can use destructuring to extract the prefix variable from the config file, and the token as well while you're at it.
+Before doing anything else, make a property to store the prefix you've configured. Instead of `const config = ...`, you can use destructuring to extract the prefix variable from the config file and the token as well while you're at it.
```

Comment:
```suggestion
Before doing anything else, make a property to store the prefix you've configured. Instead of `const config = ...`, you can use destructure the config file to extract the prefix and token variables.
```